tap_spreadsheets_anywhere/configuration.py

```diff
@@ -6,6 +6,8 @@
 LOGGER = logging.getLogger(__name__)
 
 CONFIG_CONTRACT = Schema({
+    Optional('s3_stage_bucket'): str,
+    Optional('s3_arn_role'): str,
```

Comment:
*note:* so far stage bucket and arn role configs are global for all tables. We can have multiple plugin configs so it means we can split and have for example different arn_role for different sources.